### PR TITLE
Set template settings in one single place

### PR DESCRIPTION
**Pull request by se-bastiaan**
### Summary
Set template settings in one single place

### How to test
Steps to test the changes you made:
1. Cannot really test this except for checking that it works without debug mode

### CODE REVIEW

File: website/thaliawebsite/settings/settings.py [hunk 1]
```diff
@@ -247,6 +256,7 @@
 
 # Compressor settings
 COMPRESS_ENABLED = True
+COMPRESS_OFFLINE = not DEBUG
```
Comment (pingiun) [inline website/thaliawebsite/settings/settings.py:259]: This line can be removed as it's the default value

File: website/thaliawebsite/settings/settings.py [hunk 2]
```diff
@@ -130,11 +127,23 @@
...
                 "thaliawebsite.context_processors.thumbnail_sizes",
                 "thaliawebsite.context_processors.lustrum_styling",
-            ],
+            ]
         },
     },
 ]
 
+if not DEBUG:
+    del TEMPLATES[0]["APP_DIRS"]
```
Comment (pingiun) [inline website/thaliawebsite/settings/settings.py:136]: I think it's better to do something like:

```python
templates_base = {dict with base options}
templates_debug = {dict with debug options}
templates_production = {dict with production options}

if debug:
     TEMPLATES = [ { **templates_base, **templates_debug } ]
else:
    TEMPLATES = [ { **templates_base, **templates_production } ]
```

File: website/thaliawebsite/settings/settings.py [hunk 3]
```diff
@@ -33,10 +33,7 @@
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
```
Comment (pingiun) [inline website/thaliawebsite/settings/settings.py:34]: Isn't `DEBUG` true in this whole file because it's defined here?

**Review by pingiun — CHANGES_REQUESTED** [review]

File: website/thaliawebsite/settings/settings.py (hunk 1 above)
Comment (se-bastiaan) [reply website/thaliawebsite/settings/settings.py:259]: That is not true, the default value is `False`: https://django-compressor.readthedocs.io/en/stable/settings/#django.conf.settings.COMPRESS_OFFLINE

File: website/thaliawebsite/settings/settings.py (hunk 3 above)
Comment (se-bastiaan) [reply website/thaliawebsite/settings/settings.py:34]: yes that could be what happens

**Comment by se-bastiaan** [comment]
Think we should create an issue to do this better

**Pull request closed without merging**